src/test/java/com/wikia/webdriver/common/templates/NewTestTemplateCore.java

```diff
@@ -1,36 +1,36 @@
 package com.wikia.webdriver.common.templates;
 
+import java.io.File;
+import java.lang.reflect.Method;
+
+import net.lightbody.bmp.proxy.ProxyServer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.lightbody.bmp.proxy.ProxyServer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)